lab1.js

```diff
@@ -132,12 +134,25 @@ var tooHungryDay;
  pondering protein supplements (the first day the average dips below 4
  meals)
 */
+var tooHungryDay = 0;
+var mealsPerDay = [5, 4, 3, 6, 2, 4, 3, 4, 5, 1];
 
-// your code goes here
+for (var i = 0; i < mealsPerDay.length; i++) {
+
+  var sum = 0;
+  for (var c = 0; c <= i; c++) {
+    sum = sum + mealsPerDay[c];
+  }
```

Comment:
Using a `for` loop in this way to calculate the sum is unnecessary as you are actually instantiating `var = sum` and executing this loop within a loop every time your run though the outer loop. If you debug using a `console.log` statement within the sum loop, you'll see that it is needlessly built up again and again.